identifiers/country-bd/Jatiya_Sangsad_electorates.csv

```diff
@@ -0,0 +1,301 @@
+ocd-division/country:bd/ed:mymensingh_7,Mymensingh-7
+ocd-division/country:bd/ed:mymensingh_8,Mymensingh-8
+ocd-division/country:bd/ed:mymensingh_9,Mymensingh-9
+ocd-division/country:bd/ed:mymensingh_10,Mymensingh-10
+ocd-division/country:bd/ed:mymensingh_11,Mymensingh-11
+ocd-division/country:bd/ed:jamalpur_1,Jamalpur-1
+ocd-division/country:bd/ed:jamalpur_2,Jamalpur-2
+ocd-division/country:bd/ed:jamalpur_3,Jamalpur-3
+ocd-division/country:bd/ed:jamalpur_4,Jamalpur-4
+ocd-division/country:bd/ed:jamalpur_5,Jamalpur-5
+ocd-division/country:bd/ed:netrokona_1,Netrokona-1
+ocd-division/country:bd/ed:netrokona_2,Netrokona-2
+ocd-division/country:bd/ed:netrokona_3,Netrokona-3
+ocd-division/country:bd/ed:netrokona_4,Netrokona-4
+ocd-division/country:bd/ed:netrokona_5,Netrokona-5
+ocd-division/country:bd/ed:sherpur_1,Sherpur-1
+ocd-division/country:bd/ed:sherpur_2,Sherpur-2
+ocd-division/country:bd/ed:sherpur_3,Sherpur-3
+ocd-division/country:bd/ed:sylhet_1,Sylhet-1
+ocd-division/country:bd/ed:sylhet_2,Sylhet-2
+ocd-division/country:bd/ed:sylhet_3,Sylhet-3
+ocd-division/country:bd/ed:sylhet_4,Sylhet-4
+ocd-division/country:bd/ed:sylhet_5,Sylhet-5
+ocd-division/country:bd/ed:sylhet_6,Sylhet-6
+ocd-division/country:bd/ed:habiganj_1,Habiganj-1
+ocd-division/country:bd/ed:habiganj_2,Habiganj-2
+ocd-division/country:bd/ed:habiganj_3,Habiganj-3
+ocd-division/country:bd/ed:habiganj_4,Habiganj-4
+ocd-division/country:bd/ed:maulvibazar_1,Maulvibazar-1
+ocd-division/country:bd/ed:maulvibazar_2,Maulvibazar-2
+ocd-division/country:bd/ed:maulvibazar_3,Maulvibazar-3
+ocd-division/country:bd/ed:maulvibazar_4,Maulvibazar-4
+ocd-division/country:bd/ed:sunamganj_1,Sunamganj-1
+ocd-division/country:bd/ed:sunamganj_2,Sunamganj-2
+ocd-division/country:bd/ed:sunamganj_3,Sunamganj-3
+ocd-division/country:bd/ed:sunamganj_4,Sunamganj-4
+ocd-division/country:bd/ed:sunamganj_5,Sunamganj-5
+ocd-division/country:bd/ed:chittagong_1,Chittagong-1
+ocd-division/country:bd/ed:chittagong_2,Chittagong-2
+ocd-division/country:bd/ed:chittagong_3,Chittagong-3
+ocd-division/country:bd/ed:chittagong_4,Chittagong-4
+ocd-division/country:bd/ed:chittagong_5,Chittagong-5
+ocd-division/country:bd/ed:chittagong_6,Chittagong-6
+ocd-division/country:bd/ed:chittagong_7,Chittagong-7
+ocd-division/country:bd/ed:chittagong_8,Chittagong-8
+ocd-division/country:bd/ed:chittagong_9,Chittagong-9
+ocd-division/country:bd/ed:chittagong_10,Chittagong-10
+ocd-division/country:bd/ed:chittagong_11,Chittagong-11
+ocd-division/country:bd/ed:chittagong_12,Chittagong-12
+ocd-division/country:bd/ed:chittagong_13,Chittagong-13
+ocd-division/country:bd/ed:chittagong_14,Chittagong-14
+ocd-division/country:bd/ed:chittagong_15,Chittagong-15
+ocd-division/country:bd/ed:chittagong_16,Chittagong-16
+ocd-division/country:bd/ed:rangamati,Rangamati
+ocd-division/country:bd/ed:bandarban,Bandarban
+ocd-division/country:bd/ed:khagrachari,Khagrachari
+ocd-division/country:bd/ed:coxs_bazar_1,Cox s Bazar-1
```

Comment:
It's okay to have an apostrophe within the name column. Please add here and the following lines.

```suggestion
ocd-division/country:bd/ed:coxs_bazar_1,Cox's Bazar-1
```

Screenshot from website:
![Screenshot 2023-10-25 at 8 10 20 AM](https://github.com/opencivicdata/ocd-division-ids/assets/10860174/fa6d48c2-2425-4610-bd16-e27e5db6cfaf)